client/components/VotingButtons.jsx

```diff
@@ -8,19 +8,25 @@ function Voting({ cat, dog }) {
   const dispatch = useDispatch()
 
   // win 2 points
-  // draw 1 point
   function handleSubmit(winnerId) {
-    if (winnerId !== 'skip') {
-      postVotes(winnerId)
-        .then(() => {
-          dispatch(fetchTwoPets())
-        })
-        .catch((err) => {
-          console.log(err)
-        })
-    } else {
-      dispatch(fetchTwoPets())
-    }
+    postVotes(winnerId)
+      .then(() => {
+        dispatch(fetchTwoPets())
+      })
+      .catch((err) => {
+        console.log(err)
+      })
+  }
+
+  // draw 1 point
+  function handleTie(catId, dogId) {
+    postVotesTie(catId, dogId)
+      .then(() => {
+        dispatch(fetchTwoPets())
+      })
+      .catch((err) => {
+        console.log(err)
```

Comment:
this catch isn't covered as well - whenever you have a promise remember to test both cases (when it resolves and when it rejects)